forms/AvatarInput/index.js

```diff
@@ -20,15 +20,12 @@ module.exports = React.createClass({
   },
 
   getDefaultProps: function() {
-    global.ENV = global.ENV || {};
-
     return {
       options: {
         imageDim: [600, 600],
         cropRatio: 1 / 1,
         mimetype: ['image/*'],
-        services: ['CONVERT', 'COMPUTER'],
-        debug: global.ENV.TEST_MODE
```

Comment:
I don't believe that Avatar input is in NFP, however the stuff in FileInput from may still be relevant to us.